docs/getting-started/integrate/node.mdx

```diff
@@ -9,6 +9,34 @@ One of the most common usages of Mock Service Worker in Node is utilizing our re
 
 It's recommended to configure API mocking as a part of your tests teardown, so that your tests don't have to reference any mocking during their runs, focusing on testing what matters.
 
+## Configure server
+
+Let's create a file in our mock definition directory (`src/mocks`) where we would configure our request mocking server.
+
+<Action>
+  Create a <code>src/mocks/server.js</code> file:
+</Action>
+
+```bash
+$ touch src/mocks/server.js
+```
+
+In the `server.js` file we are going to create a server instance with our request handlers defined earlier.
+
+<Action>
+  Import <code>setupServer</code> function from the <code>msw/node</code> package and
```

Comment:
This is not *strictly* accurate, as the "package" is still `msw` and we're just importing from the `/node` entrypoint, but I didn't want to clutter this simple step with formalities.

Reply:
I'd keep it as `...from the <code>msw</code> package`. So we don't tight the action points to the implementation too much (what if in the future we'd expose `setupServer` from a different place?).

Reply:
Agreed, not sure what I was thinking there. 🙃